docs/variants/msi_z690/community/community-hcl.md

```diff
@@ -0,0 +1,31 @@
+# Community HCL - MSI Z690-A WIFI DDR4
+
+## About
+
+The following document contains information about hardware compatible
+with the MSI Z690-A WIFI DDR4 platform with Dasharo firmware.
+
+The following list is created based on the information from the
+community - people which are currently using the platform with Dasharo
+firmware.
+
+This document contains three main sections:
+- `Testing method` in which we are describing our preferred testing method.
+- `Hardware Compatible List` in which in a labelled manner we show all
+    compatible hardware.
+- `Support us` - in which we describe the method of how it is possible to
+    contribute to the development of this list.
+
+## Testing method
+
+## Hardware Compatible List
+
+| No.  | Supported hardware                                | Link to description | Notes               |
+|:-----|:--------------------------------------------------|:-------------------:|:-------------------:|
+| 1.   |                                                   |                     |                     |
+
+## Support Us
+
+Hardware compatible with Dasharo firmware you might report by:
+- using `Dasharo Issues project`, which is available under this
+    [link](https://github.com/Dasharo/dasharo-issues/issues).
```

Comment:
@sulewskiprzemyslaw we need procedur step by step for community what to do, from above description it is not clear to me what are the steps. Reporting and handling reports also could be documented and as much as possible simplified. Here we rather have concept draft then working operation procedure.

Some requirements, which I though are obvious:
* We should explain in details what are the sources of community HCL and test results.
* We should always link original report if it was on some forum/mailing list/matrix channel or IRC.
* Community test results and HCL should be availble in the same location (maybe on different tab, or different page sction) as 3mdeb test results. 
* Format of test results/HCL presentation should be exactly the same. 
* We should report community test results/HCL in newsletter.
* We should inform how we treat contribution, that we give kudos for those who want as they want, if they don't want we will just say this is anonymous report.